### PR TITLE
test | Aditi | color reduce work

### PR DESCRIPTION
**Pull request by aditi-tiwari17**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/onboarding/onboarding.component.scss [hunk 1]
```diff
@@ -398,7 +398,7 @@
     #sFform-xZyf input,
     #sFform-xZyf textarea,
     #sFform-xZyf select {
-        border: 1px solid $bright-grey !important;
+        border: 1px solid #525265 !important;
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/onboarding/onboarding.component.scss:401]: add color var

File: apps/web-giddh/src/app/onboarding/onboarding.component.scss [hunk 2]
```diff
@@ -408,11 +408,11 @@
     }
 
     #sFformfield-xZyf label {
-        color: $light-grayish-navy-blue !important;
+        color: #73737f !important;
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/onboarding/onboarding.component.scss:411]: add color var

File: apps/web-giddh/src/app/onboarding/onboarding.component.scss [hunk 3]
```diff
@@ -408,11 +408,11 @@
...
     }
 
     #sFSection-Jkvq .sFsubmit.btn-primary {
-        background: $deep-green !important;
+        background: #28ab00 !important;
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/onboarding/onboarding.component.scss:415]: add color var

File: apps/web-giddh/src/assets/css/color.scss [hunk 4]
```diff
@@ -68,172 +59,56 @@ $cinnabar-orange: #df4927;
...
-$granny-apple: #c9e6ca;
-$spun-pearl: #b1b1b3;
-$gray-red: #9b9b9b;
-$dark-gray-red: #2c2c2c;
-$extra-dark-orange: #e34b26;
-$selago: #fefdfe;
-$bright-grey: #525265;
-$light-grayish-navy-blue: #73737f;
-$valspar-paint-autumn-fog: #bbbbbb;
+$cactus: #587058;
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/assets/css/color.scss:108]: I have seen some of color var added only on color.scss file and not using on other css file. find Below to name of color var and just make sure if any of the color is not using on css then please remove that color on color.scss file.

$burgundy,  $dull-pink, $atlantis , $violet-red, $java, $bus-yellow , $cactus, $cornflower-blue-dark, $pale-rose, $cloud-burst, $deep-green-cyan, $granny-apple, $chablis

Comment (aditi-tiwari17) [reply apps/web-giddh/src/assets/css/color.scss:108]: @shalinee-walkover  These colors are added in tb-pl-bs, onboarding and in design filters.

File: apps/web-giddh/src/app/onboarding/onboarding.component.scss (hunk 1 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/onboarding/onboarding.component.scss]: done

File: apps/web-giddh/src/app/onboarding/onboarding.component.scss (hunk 2 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/onboarding/onboarding.component.scss]: done

File: apps/web-giddh/src/app/onboarding/onboarding.component.scss (hunk 3 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/onboarding/onboarding.component.scss]: done

**Review by shalinee-walkover — APPROVED** [review]

**Pull request merged**